CONTRIBUTING.md

```diff
@@ -190,15 +190,40 @@ that can help you.
 
 ## Content contributors
 
-### Creating a news post
+### Creating a blog post
 
-*Coming soon! We are still figuring out exactly how this will look*
+Blog posts must be created in the ``conda-dot-org/blog`` directory with a file naming convention of ``YYYY-MM-DD-title.md``. 
 
+Let's say you're announcing a cool new project you've recently released. Your file would be named something like ``2023-04-22-cool-project-v1-release.md``.
 
-### Creating a blog post
+conda.org blog posts support the following metadata:
 
-*Coming soon! We are still figuring out exactly how this will look*
+```
+title: The title of your blog.
+slug: The name of your blog page within the conda.org url.
+tags: Tags to group your blog with others and enable better filtering and browsing (e.g. conda, release, package, announcement).
+tease: (Optional) A short summary of or exceprt from your blog. If this is not supplied, conda.org will use the first 20 words of your blog post.
```

Comment:
I can't remember if "20 words" is correct, or if it's a certain number of characters.

Reply:
20 words. 20 characters would be barely be a couple of words! 😅